doc/examples/rendering/macbethChart.gfr

```diff
@@ -122,11 +116,16 @@ __children["Outputs"]["outputs"]["output1"]["name"].setValue( 'Interactive/Beaut
 __children["Outputs"]["outputs"]["output1"]["fileName"].setValue( 'beauty' )
 __children["Outputs"]["outputs"]["output1"]["type"].setValue( 'ieDisplay' )
 __children["Outputs"]["outputs"]["output1"]["data"].setValue( 'rgba' )
+__children["Outputs"]["outputs"]["output2"]["name"].setValue( 'Batch/Beauty' )
+__children["Outputs"]["outputs"]["output2"]["fileName"].setValue( '${HOME}/Desktop/test.exr' )
```

Comment:
I think in the other examples we've kept renders under `${project:rootDirectory}/renders/...` and tried to avoid collisions. Be aware that as the script hasn't been saved `${scriptName}` is not reliable.

Reply:
Good catch. Knew I forgot something...

Reply:
ac438208f30ae4cbff68fce10b518746a06c4c3e

Reply:
ac438208f30ae4cbff68fce10b518746a06c4c3e

> Be aware that as the script hasn't been saved ${scriptName} is not reliable.

We're using it in the Wedge Tests example too, and the ImageReader node fails on it. Should we look into some alternate solution for renders from examples? Maybe an `_examples` directory?

Reply:
Humm, I'm sure I pushed a PR that fixed that, to make sure it worked without saving.... https://github.com/GafferHQ/gaffer/commit/f71c811e83b1be2428ad364a3fa23ab34d477a5a, oh rubbish, I must have missed the wedge one, good catch, thanks!

Reply:
fwiw, I think it only matters if the downstream script needs to read them back in, as you encounter variable expansion of `${scriptName}` in the dispatched execution .vs. unsaved session. So for examples that don't need to reference their own renders it's probably ok to use `${scriptName}` for simplicity. I like the idea of an `examples` directory though, keeps it all together..